init.lua

```diff
@@ -122,6 +122,14 @@ local function node_is_plant(node)
 	if minetest.registered_nodes[node.name].groups.flora == 1 then
 		return true
 	end
+	
+	if minetest.registered_nodes[node.name].groups.leaves == 1 then
```

Comment:
At this point it would be better to put `minetest.registered_nodes[node.name].groups` into a separate group for easier access. This will even improve performance due to fewer table lookups.

Reply:
How would I implement that?

Reply:
Simple as
```Lua
local groups = minetest.registered_nodes[node.name].group

if groups.whatever == 1 then
...
```